movement/io/load_poses.py

```diff
@@ -696,3 +709,118 @@ def _ds_from_valid_data(data: ValidPosesDataset) -> xr.Dataset:
             "ds_type": "poses",
         },
     )
+
+
+def from_anipose_style_df(
+    df: pd.DataFrame,
+    fps: float | None = None,
+    individual_name: str = "individual_0",
+) -> xr.Dataset:
+    """Create a ``movement`` poses dataset from an Anipose 3D dataframe.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Anipose triangulation dataframe
+    fps : float, optional
+        The number of frames per second in the video. If None (default),
+        the ``time`` coordinates will be in frame units.
+    individual_name : str, optional
+        Name of the individual, by default "individual_0"
+
+    Returns
+    -------
+    xarray.Dataset
+        ``movement`` dataset containing the pose tracks, confidence scores,
+        and associated metadata.
+
+
+    Notes
+    -----
+    Reshape dataframe with columns keypoint1_x, keypoint1_y, keypoint1_z,
+    keypoint1_confidence_score,keypoint2_x, keypoint2_y, keypoint2_z,
+    keypoint2_confidence_score...to array of positions with dimensions
+    time, individuals, keypoints, space, and array of confidence scores
+    with dimensions time, individuals, keypoints.
```

Comment:
This is the old dimension order, please update to the current one.

 ```suggestion
     time, space, keypoints, individuals, and array of confidence scores
     with dimensions time, keypoints, individuals.